Rakefile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# make continuous integration using rubygem-packaging_rake_tasks
+# Copyright © 2015 SUSE
+# MIT license
+
+require "packaging/tasks"
+require "packaging/configuration"
+# skip 'tarball' task, it's redefined here
+Packaging::Tasks.load_tasks(:exclude => ["tarball.rake"])
+
+Packaging.configuration do |conf|
+  conf.obs_project    = "YaST:Head"
+  conf.obs_sr_project = "openSUSE:Factory"
+  conf.package_dir    = ".obsdir"
+  conf.skip_license_check << /.*/
+end
```

Comment:
ci.opensuse.org do not by default submit to SP2 but to factory. If you are interested, ci.s.d send to SP2